unittests/files/autocomplete.ts

```diff
@@ -0,0 +1,59 @@
+// @ts-expect-error should be a string
+document.body.getElementsByTagName("input")[0].autocomplete = false;
+// @ts-expect-error should not be empty
+document.body.getElementsByTagName("input")[0].autocomplete = "";
```

Comment:
Looks like it can be empty: https://github.com/web-platform-tests/wpt/blob/606530b23a2ed89e965403475c6f40a98afb0900/html/semantics/forms/the-form-element/form-autocomplete.html#L63-L67

And also per the step 32 of the spec: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-processing-model

>Default: Let the element's [IDL-exposed autofill value](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#idl-exposed-autofill-value) be the empty string, and its [autofill hint set](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-hint-set) and [autofill scope](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-scope) be empty.